### PR TITLE
docs: upload 1.7.0 changelog

### PR DESCRIPTION
**Pull request by milasuperstar**
Upgrade docs version to 1.7.0

Signed-off-by: Mila Wu <wumila1996@gmail.com>

### CODE REVIEW

**Review by boojack — APPROVED** [review]
LGTM

File: content/changelog/bytebase-1-7-0.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Bytebase 1.7.0
+author: Mila
+published_at: 2022/10/27 17:00:00
+description: "- Added 31 review rules for MySQL. - Support DDL validation in SQL review. - Support canceling gh-ost sync tasks. "
+---
+
+## 🚀 Features
+
+- Added 31 [review rules](https://www.bytebase.com/docs/sql-review/review-rules/overview) for MySQL.
```
Comment (tianzhou) [inline content/changelog/bytebase-1-7-0.md:10]: Use relative path /docs/sql-review/review-rules/overview

File: content/changelog/bytebase-1-7-0.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- Support canceling gh-ost sync tasks.
+
+## 🎄 Enhancements
+
+- Hide similar activities on the issue detail page.
+- Improved performance for creating and approving issues.
+
+## 🎠 Community
+
+- Bytebase becomes [the first and only database CI/CD project](https://www.bytebase.com/blog/cncf-landscape) to be included in the CNCF Landscape! 🎉
```
Comment (tianzhou) [inline content/changelog/bytebase-1-7-0.md:21]: Use relative path

File: content/changelog/bytebase-1-7-0.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- Thanks to @[VUHAILAM](https://github.com/VUHAILAM) for feat(sql-review): implement postgresql no duplicate columns in the index. [#3037](https://github.com/bytebase/bytebase/pull/3037)
+- Thanks to @[Zheaoli](https://github.com/Zheaoli) for helping us refactor the issue template for the Bytebase repo. [#3050](https://github.com/bytebase/bytebase/pull/3050)
+
+## 📕 Installation and Upgrade
+
+Follow https://github.com/bytebase/bytebase#installation. If you upgrade from a previous version, just restart after obtaining the new release binary.
```
Comment (tianzhou) [inline content/changelog/bytebase-1-7-0.md:27]: Use /docs/get-started/install/overview

Comment (tianzhou) [inline content/changelog/bytebase-1-7-0.md:27]: Use /docs/get-started/install/overview

**Review by tianzhou — CHANGES_REQUESTED** [review]

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**